lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -138,10 +152,17 @@ export class NativeInteractionClient extends BaseInteractionClient {
      * @param request
      * @param reqTimestamp
      */
-    protected async handleNativeResponse(response: NativeResponse, request: NativeTokenRequest, reqTimestamp: number): Promise<AuthenticationResult> {
+    protected async handleNativeResponse(response: NativeResponse, request: NativeTokenRequest, reqTimestamp: number, nativeAcquireTokenMeasurement?: InProgressPerformanceEvent): Promise<AuthenticationResult> {
```

Comment:
Since this is just for the error case, can we catch the error above instead of passing the measurement down?

Reply:
Yes, prefer not to pass measurements done, instead have them be self contained within a given function.

Reply:
So that is the q I had. @jasonnutter is there no need to retain the measurement instance (since we started it above) for an error? Should I create a new one before the throw? I was assuming it should be done, I can sync up with you on this.

Reply:
I can modify it to a try/catch ofcourse. I thought this makes it easy to read, no special preference.

Reply:
Done.